watir-rspec.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
   gem.version       = Watir::RSpec::VERSION
 
-  gem.add_dependency "rspec", "~>2.0"
+  gem.add_dependency "rspec", "~>3.0"
```

Comment:
That wasn't before the case. I'm not familiar with RSpec 3 formatters, but is that registering really needed? Can't you just configure to use custom formatter like with RSpec 2? What does that registering mean in context of RSpec?